_scripts/instruction-widget/templates/install/debian.html

```diff
@@ -12,7 +12,7 @@
 {{/devuan}}
 {{^devuan}}
 First you'll have to follow the instructions <a
-href='http://backports.debian.org/Instructions/'>here</a> to enable the Jessie
+    href='http://backports.debian.org/Instructions/'>here</a> to enable the {{#jessie}}Jessie{{/jessie}}{{#stretch}}Stretch{{/stretch}}
```

Comment:
This indentation looks strange to me. Did you do it on purpose to show that this is part of the  `<a>` tag from the previous line? If so, feel free to leave it as is, I just wanted to make sure it was intentional.